packages/react-scripts/config/jest/transform.js

```diff
@@ -1,14 +1,13 @@
-// @remove-on-eject-begin
 /**
  * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
  *
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  */
-// @remove-on-eject-end
 
-const babelDev = require('../babel.dev');
 const babelJest = require('babel-jest');
 
-module.exports = babelJest.createTransformer(babelDev);
+module.exports = babelJest.createTransformer({
+  presets: [require.resolve('babel-preset-react-app')]
```

Comment:
you don't really need this transformer any more. If you are creating a `.babelrc`, you can just use babel-jest directly by making it an explicit dependency and Jest will pick up the babelrc file.